### PR TITLE
IAB upgrade to version 4.21.0

### PR DESCRIPTION
**Pull request by giancarlobi**
@DiegoPino I checked version 4.21.0 and seems work well. This PR is from my git RC1 to esmero, probably there are some other chages, strage because before PR to my git I compared RC1 my and esmero and they were aligned.
Feel you free to make any changes or discard this PR, how is better for you/archipelago/esmero.

### CODE REVIEW

File: format_strawberryfield.libraries.yml [hunk 1]
```diff
@@ -136,13 +136,13 @@ pdfs_strawberry:
...
 mirador_projectmirador:
-  version: 3.0.0-rc.5
+  version: 3.0.0
   license:
     name: Apache
     url: //github.com/ProjectMirador/mirador/blob/master/LICENSE
     gpl-compatible: true
   js:
-    https://cdn.jsdelivr.net/npm/mirador@3.0.0-rc.5/dist/mirador.min.js: { external: true, minified: true, preprocess: false}
+    https://cdn.jsdelivr.net/npm/mirador@3.0.0/dist/mirador.min.js: { external: true, minified: true, preprocess: false}
```
Comment (DiegoPino) [inline format_strawberryfield.libraries.yml:145]: This here is already in ISSUR-102 and its own issue . Can merge of course but I will need to rebase. Will let you know

File: src/Plugin/Field/FieldFormatter/StrawberryMediaFormatter.php [hunk 2]
```diff
@@ -279,9 +279,11 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
                 $elements[$delta]['media'.$i]['#attached']['drupalSettings']['format_strawberryfield']['openseadragon'][$uniqueid]['dr:uuid']  = $file->uuid();
                 // Used to keep annotations around during edit.
                 // Note: Since View modes are cached, if no change to the NODE this will be served from a cache! mmm.
-                $elements[$delta]['media'.$i]['#attached']['drupalSettings']['format_strawberryfield']['openseadragon'][$uniqueid]['keystoreid'] = WebAnnotationController::getTempStoreKeyName($fieldname,$delta, $nodeuuid);
-                $elements[$delta]['media'.$i]['#attached']['drupalSettings']['format_strawberryfield']['openseadragon'][$uniqueid]['webannotations'] = (boolean)$webannotations;
-                $elements[$delta]['media'.$i]['#attached']['drupalSettings']['format_strawberryfield']['openseadragon'][$uniqueid]['webannotations_tool'] = $webannotations_tool ? $webannotations_tool : 'rect';
+                if ($this->currentUser->hasPermission('view strawberryfield webannotation')) {
```
Comment (DiegoPino) [inline src/Plugin/Field/FieldFormatter/StrawberryMediaFormatter.php:282]: I will discard this because ISSUE-102 has a better option for this.

Comment (giancarlobi) [reply src/Plugin/Field/FieldFormatter/StrawberryMediaFormatter.php:282]: ok, ok for me, I think some spurious pr remain into my RC1

**Review by DiegoPino — APPROVED** [review]
Will merge as it is and rebase my own... do not like to touch other people's commits!

**Pull request merged**